index.html

```diff
@@ -0,0 +1,214 @@
+    <div class="work_description">
+      <h2>Tonic</h2>
+      <ul class="conduct">
+        <li class="conduct_head">CANOPY</li>
+        <li><img src="./image/Counter.svg" alt=""></li>
+        <li class="conduct_para">Back End Dev</li>
+        <li><img src="./image/Counter.svg" alt=""></li>
+        <li class="conduct_year">2015</li>
+      </ul>
+      <p class="work_paragraph">A daily selection of privately personalized reads; 
+        no accounts or sign-ups required.</p>
+      <ul class="work_languages">
+       <li>html</li>
+       <li>css</li>
+       <li>javascript</li>
+      </ul>
+        <button class="btn">See Project</button>
+    </div>
+
+</article>
+    </div>
+</section>
+<section class="about">
+<article class="about_head">
+  <h2>About
+    Myself</h2>
+  <p>Hello I’m a software developer! 
+    I can help you build a product , 
+    feature or website Look through 
+    some of my work and experience! 
+    If you like what you see and have 
+    a project you need 
+    coded, don’t hestiate to contact me.</p>
+  <a href="">LET’S CONNECT</a>
+  <ul>
+    <li class="item1"><a href=""><img src="./image/Vectorgithub.svg" alt=""></a></li>
+    <li class="item1"><a href=""><img src="./image/Linkedin iconlinkedin.svg" alt=""></a></li>
+    <li class="item1"><a href=""><img src="./image/Vectormedium.svg" alt=""></a></li>
+    <li class="item1"><a href=""><img src="./image/Vectortwitter (1).svg" alt=""></a></li>
+    <li class="item1"><a href=""><img src="./image/Vector (3).svg" alt=""></a></li>
+  </ul>
+  <button class="btn">See Project</button>
+</article>
+<article class="about_language">
+<div class="item">
+  <h3>Languages</h3>
+  <img src="image/down.svg" alt="">
+</div>
+<ul class="about_lang">
+   <li>
+     <img src="image/javascript.svg" alt="">
+     <p>javascript</p>
+   </li>
+   <li>
+    <img src="image/html (2).svg" alt="">
+    <p>html</p>
+   </li>
+   <li>
+    <img src="image/css.svg" alt="">
+    <p>css</p>
+   </li>
+</ul>
+<li class="item">
+  <h3>Framework</h3>
+  <img src="image/left2.svg" alt="">
+
+</li>
+<hr>
+
+<li class="item">
+  <h3>Skills</h3>
+  <img src="image/left2.svg" alt="">
+</li>
+<hr>
+</article>
+  </section>
+  <section class="contact">
+    <article class="contact_footer">
+      <h1>contact me</h1>
+      <img src="image/contact form shapes-mobile@2x.svg" alt="">
+      <p>If you have an application you are interested
+         in developing, a feature that you need built or a project
+          that needs coding. I’d
+         love to help with it </p>
+         <form class="form-start" action="">
+          <input type="name" type="text" name="name" class="name" placeholder="write your name">
+          <input type="email" type="email" name="email" class="email" placeholder="write your email">
+          <input type="message" type="text" name="message" class="message" placeholder="write your message">
+          <button class="start">get in touch</button>
```

Comment:
- [ ] Kindly try to stick to the figma design as much as possible to make your project looks good. try to leverage padding colors etc.

Your design:
![image](https://user-images.githubusercontent.com/90524466/207955639-f27509ec-948a-4b86-a087-b1be4ccd751f.png)

Figma design:
![image](https://user-images.githubusercontent.com/90524466/207955821-9b5d352f-210b-48dc-8203-1c88021df718.png)